src/status_im/ui/components/qr_code_viewer/views.cljs

```diff
@@ -3,19 +3,25 @@
             [status-im.react-native.js-dependencies :as rn-dependencies]
             [status-im.ui.components.qr-code-viewer.styles :as styles]
             [status-im.ui.components.react :as react]
-            [status-im.ui.components.svg :as svg])
-  (:require-macros [status-im.utils.views :refer [defview letsubs]]))
+            [status-im.ui.components.svg :as svg]))
 
 (defn qr-code [{:keys [size value]}]
   (let [uri (reagent/atom nil)]
-    (.toString rn-dependencies/qr-code value #(reset! uri %2))
+    (.toString
+     rn-dependencies/qr-code
+     value
+     #js {:margin 0 :width size}
```

Comment:
I think we tend to use clj->js instead.

Reply:
no,  why ? #js is better

Reply:
Understood. There is also `https://github.com/mfikes/cljs-bean` in `deps.edn` committed already. Which is supposed to be faster than `clj->js`. I think we're going to move over to `->js` and `->clj` provided by `cljs-bean` everywhere sometimes perhaps soon. Shall I use it instead?

Reply:
yes please

Reply:
👍 going to use `->js` and `->clj` for js interop going forward

Reply:
@flexsurfer i somehow missed `#js` usage across our codebase :( thanks @bitsikka for reminding about `cljs-bean`. Good stuff :)

Reply:
did it.

Indeed good stuff :)
huge fan of mfikes's work, and every ounce of perf improvement matters.

have to really check doc/source though, maybe not needed for #js <col> 🤔 because cljs transpiler might already automatically keep that as native js data-structure and there myght be no conversion of ds involved.

I have to do some long chores today.. will look into doc/source later.